Chat: describe API (#6640)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChat/1 Configuration/Configuration.md

---
##### shortDescription

> An object defining configuration properties for the Chat UI component.

Reviewer: ```suggestion
A configuration object for the Chat UI component.
```
